src/main/java/frc/robot/Constants.java

```diff
@@ -27,7 +27,8 @@ public AimModes previous() {
     },
     EJECT(2500.0, 0.0),
     LAUNCH(0.0, 0.0),
-    TARMAC(2530.0, 0.0),
+    TARMAC(2480.0, 0.0),
+    ATARMAC(2480.0, 0.0),
```

Comment:
atarmac and tarmac are the same.  Were they meant to be different?

Reply:
We added atarmac and tarmac as a way to change the auto ball speed and tele ball speed independently, as they are shooting differently somehow. This is part of a larger issue tracked as #224

Reply:
But they arent different is my point.  So, wondering if you have a mistake here, and thought you were adjusting them differently...but weren't

Reply:
No, it's open to give us options should something go wrong, resolving this